app/services/proofing/socure/reason_codes/api_client.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Socure
+    module ReasonCodes
+      class ApiClient
+        class ApiClientError < StandardError; end
+
+        def download_reason_codes
+          http_response = make_reason_code_http_request
+          http_response.body['reasonCodes']
+        rescue Faraday::ConnectionFailed,
+               Faraday::ServerError,
+               Faraday::SSLError,
+               Faraday::TimeoutError,
+               Faraday::ClientError,
+               Faraday::ParsingError => e
+          raise ApiClientError, e.message
```

Comment:
I'm inferring that the idea here is that if the request fails, it's not a big deal because we can either manually re-run it, or just get it next week, versus implementing lots of retry logic? (I'm fine with that, just confirming the plan.)

Reply:
Yeah, exactly